templates/legal/shared/_appendix-support-scope.md

```diff
@@ -0,0 +1,439 @@
+<h1 id="appendix-support-scope">Appendix {{number}} - Support scope</h1>
```

Comment:
This has gone from this:

![image](https://user-images.githubusercontent.com/519935/50825229-6e378100-1330-11e9-8884-715e20d47df6.png)

To this:

![image](https://user-images.githubusercontent.com/519935/50825250-78f21600-1330-11e9-9190-2c31c11812bd.png)

That feels like a regression. You may need to add some more explicit markup here to make the spacing better. Although it's a shame that Vanilla doesn't treat `<h1>` followed by `<h2>` better.